DQMOffline/Alignment/src/DiMuonMassBiasClient.cc

```diff
@@ -186,18 +186,18 @@ diMuonMassBias::fitOutputs DiMuonMassBiasClient::fitVoigt(TH1* hist, const bool&
 
   RooAddPdf fullModel("fullModel", "Signal + Background Model", RooArgList(voigt, expo), RooArgList(s, b));
   if (fitBackground_) {
-    fullModel.fitTo(datahist, RooFit::PrintLevel(-1), RooFit::Save(), RooFit::Range(xmin, xmax));
```

Comment:
what if for whatever reason, one decides to actually use a restricted range?

Reply:
Then you just add it back, checking before that the issue in RooFit where the `Range()` code path is unreasonably slow in this fit was fixed:
https://github.com/root-project/root/issues/11637

I suggest to remove the redundant Range argument now in CMSSW for two reasons:
* even after I fixed the problem in ROOT master, ranged fits will take more computation time then non-ranged fits, because RooFit calculates projection coefficients to project the RooAddPdf coefficients from the subrange to the full range. You probably want to avoid this overhead in CMSSW
* quick fix for the ROOT master builds, because I won't have time to fix the actual issue in upstream ROOT so soon

Reply:
Oh or even better you could just change the definition range of the observable

Reply:
> or even better you could just change the definition range of the observable

that way the frame won't contain all the possible input data (one might want to look outside of the fit range).